scipy/stats/tests/test_distributions.py

```diff
@@ -2888,6 +2888,66 @@ def test_minimum_data_zero_gh17801(self):
             _assert_less_or_close_loglike(dist, data)
 
 
+class TestPowerLogNorm:
+
+    # reference values were computed via mpmath
+    # from mpmath import mp
+    # mp.dps = 80
+    # def powerlognorm_sf_mp(x, c, s):
+    #     x = mp.mpf(x)
+    #     c = mp.mpf(c)
+    #     s = mp.mpf(s)
+    #     return mp.ncdf(-mp.log(x) / s)**c
+    #
+    # def powerlognormal_cdf_mp(x, c, s):
+    #     return 1 - powerlognorm_sf_mp(x, c, s)
```

Comment:
Oops, my bad.
```suggestion
    #     return mp.one - powerlognorm_sf_mp(x, c, s)
```
No harm done because no Python arithmetic happens with the 1, but better form to use `mp.one`.